code/modules/hydroponics/spreading/spreading_growth.dm

```diff
@@ -143,6 +143,8 @@
 		if(!istype(check_turf))
 			continue
 		for(var/obj/effect/vine/neighbor in check_turf.contents)
+			if(QDELETED(neighbor))
```

Comment:
Should comment on what's going on here (presumably some kind of deletion loop), as it looks wrong at a glance.

Reply:
It's a refresh proc to wake up vines that may have fallen asleep. Not actually sure if the loop needs to happen at all.